index.html

```diff
@@ -196,7 +203,7 @@ <h4>About me</h4>
           like what you see and have a project you need coded, don’t hesitate to
           contact me.
         </p>
-        <button type="button" class="cards-btn">Get My Resume</button>
+        <button type="button" class="cards-btn" >Get My Resume</button>
       </div>
       <hr class="divider" />
       <div class="technologies">
```

Comment:
- [ ] You should minimize the space between the list items to make it look like in the Figma design as the previous reviewer has suggested. You can try something like:
Removing  `**min-height: 50vh;**`  from above
```
.lang ul {
    display: flex;
    flex-direction: column;
    justify-content: flex-start;
}
```
Adding the following to the **.technologies ul**
```
 .technologies ul {
    justify-content: flex-start;   
}
```